ThemeDecorator/ThemeDecorator.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * Exports the {@link my-theme/ThemeDecorator.ThemeDecorator} HOC
+ *
+ * @module my-theme/ThemeDecorator
+ */
+
+import {addAll} from '@enact/core/keymap';
+import classnames from 'classnames';
+import hoc from '@enact/core/hoc';
+import React from 'react';
+import {ResolutionDecorator} from '@enact/ui/resolution';
+import {FloatingLayerDecorator} from '@enact/ui/FloatingLayer';
+import SpotlightRootDecorator from '@enact/spotlight/SpotlightRootDecorator';
+
+import Skinnable from '../Skinnable';
+
+import screenTypes from './screenTypes.json';
+import css from './ThemeDecorator.module.less';
+
+/**
+ * Default config for {@link my-theme/ThemeDecorator.ThemeDecorator}.
+ *
+ * @memberof my-theme/ThemeDecorator
+ * @hocconfig
+ */
+const defaultConfig = {
+	float: true,
+	noAutoFocus: false,
+	ri: {
+		screenTypes
+	},
+	spotlight: true,
+	skin: true
+};
+
+/**
+ * {@link my-theme/ThemeDecorator.ThemeDecorator} is a Higher-order Component that applies
+ * MyTheme theming to an application. It also applies
+ * [floating layer]{@link ui/FloatingLayer.FloatingLayerDecorator},
+ * [resolution independence]{@link ui/resolution.ResolutionDecorator},
+ * [custom text sizing]{@link my-theme/ThemeDecorator.TextSizeDecorator},
```

Comment:
This isn't included in the ThemeDecorator (nor is I18nDecorator though I assume it will be shortly so okay to leave that imo).